File: 4247-minimum-capacity-box/minimum-capacity-box.py
class Solution:
    def minimumIndex(self, capacity: list[int], itemSize: int) -> int:
        n = len(capacity)
        cap = 101
        idx = -1
        for i in range (0,n) :
            if capacity[i] >= itemSize and capacity[i] < cap :
                cap = capacity[i]
                idx = i    
        return idx

File: 4247-minimum-capacity-box/minimum-capacity-box.cpp
class Solution {
public:
    int minimumIndex(vector<int>& capacity, int itemSize) {
        int idx = -1;
        for(int i = 0; i< capacity.size(); i++) {
            if(capacity[i] >= itemSize) {
                if(idx == -1) idx = i;
                else if (capacity[i] < capacity[idx]) {
                    idx = i;
                }
            }
        }
        return idx;
    }
};